packages/transmute-framework/README.md

```diff
@@ -1 +1,166 @@
-# Transmute Framework
\ No newline at end of file
+# Transmute Framework
+
+The transmute framework converts javascript objects to ipfs hashes, and stores them on ethereum smart contracts.
+
+It is meant to be a building block for decentralized applications that are built on immutable event logs.
+
+It also supports some Redux like functionality which makes building models from event streams easy. These models can then be saved to document databases for querying and indexing.
+
+The environment used by the framework can easily be configured in `./src/transmute-config/env.json`.
+
+#### Getting Started With Ganache
+
+If you want to use the framework with ganache and a local ipfs node, you can do so by using the `localhost` env.
+
+First, make sure to install and start ipfs and ganache in the default manner:
+
+```
+ipfs daemon
+ganache-cli
+```
+
+Next, use `TRANSMUTE_ENV` to test the framework with this local environment.
+
+```
+npm i
+TRANSMUTE_ENV='localhost' npm run truffle:test
+TRANSMUTE_ENV='localhost' npm run truffle:migrate
+TRANSMUTE_ENV='localhost' npm run test
+```
+
+#### Getting Started With Minikube
+
+If you are using the minikube environment, you will need to ensure that the framework is configured to connect to the correct ipfs and ethereum rpc interface.
+
+
+Once you have setup the minikube cluster:
```

Comment:
May want to reference this => https://github.com/transmute-industries/transmute/blob/master/README.md  for minikube setup